Documentation/operations/upgrade.rst

```diff
@@ -464,6 +467,9 @@ Helm Options
 
 * Values  ``enableK8sEventHandover`` and ``enableCnpStatusUpdates`` have been removed.
 
+* Deprecated values ``strict``, ``partial``, ``probe`` and ``disabled`` for ``kubeProxyReplacement`` option have been
+  removed. Please use ``true`` and ``false`` instead.
```

Comment:
Small nit: Please use ``true`` **or** ``false`` instead.

Reply:
sorry, I don't get the above comment :thinking:. Do we have the same line here?

> Please use `true` or `false` instead

Reply:
ah ignore, I cannot read logically, it's `or` instead of `and`

Reply:
I think you missed this one :(

Reply:
let me check and send quick PR, good thing is that docs change will not require full CI run.

Reply:
thanks and done